src/fsharp/FSComp.txt

```diff
@@ -1551,6 +1552,8 @@ forFormatInvalidForInterpolated4,"Interpolated strings used as type IFormattable
 3384,scriptSdkNotDeterminedUnexpected,"The .NET SDK for this script could not be determined. If the script is in a directory using a 'global.json' then ensure the relevant .NET SDK is installed. Unexpected error '%s'."
 3384,scriptSdkNotDeterminedNoHost,"The .NET SDK for this script could not be determined. dotnet.exe could not be found ensure a .NET SDK is installed."
 3385,tcInvalidStructReturn,"The use of '[<Struct>]' on values, functions and methods is only allowed on partial active pattern definitions"
+3386,tcImplicitConversionUsed,"This expression uses an implicit conversion to convert type '%s' to type '%s'. Warnings are enabled for implicit conversions. Consider using an explicit conversion or disabling this warning."
```

Comment:
for the two new diagnostics, I think it would be helpful to specify _which_ `op_Implicit` is used (ie render the type name through the DisplayEnv so that the exact method name the user would put in) in the message itself. If it's part of the message it becomes very easy to make a codefix that inserts the expression for the user to 'solve' the warning.

Reply:
Yes agreed, thanks!